pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -28,26 +27,25 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 	networking_core "istio.io/istio/pilot/pkg/networking/core/v1alpha3"
 	authn_plugin "istio.io/istio/pilot/pkg/networking/plugin/authn"
-	"istio.io/istio/pilot/pkg/serviceregistry"
 	"istio.io/istio/pilot/pkg/serviceregistry/aggregate"
 )
 
-// InitDebug initializes the debug handlers and adds a debug in-memory registry.
+// InitDebug initializes the debug handlers.
 func (s *DiscoveryServer) InitDebug(mux *http.ServeMux, sctl *aggregate.Controller) {
-	// For debugging and load testing v2 we add an memory registry.
-	s.MemRegistry = NewMemServiceDiscovery(
-		map[model.Hostname]*model.Service{ // mock.HelloService.Hostname: mock.HelloService,
-		}, 2)
-	s.MemRegistry.EDSUpdater = s
-	s.MemRegistry.ClusterID = "v2-debug"
-
-	sctl.AddRegistry(aggregate.Registry{
-		ClusterID:        "v2-debug",
-		Name:             serviceregistry.ServiceRegistry("memAdapter"),
-		ServiceDiscovery: s.MemRegistry,
-		ServiceAccounts:  s.MemRegistry,
-		Controller:       s.MemRegistry.controller,
-	})
+	//// For debugging and load testing v2 we add an memory registry.
```

Comment:
Mem registry is useful for some large scale tests - agree it should be off by default, but still need a way to enable it 
somehow.